src/App.tsx

```diff
@@ -1,8 +1,9 @@
 import React from 'react';
 
 import './App.scss';
+import { PostList } from './components/PostList';
 
-// import postsFromServer from './api/posts';
+import postsFromServer from './api/posts';
```

Comment:
nit: Since you're importing `postsFromServer`, consider removing the commented out imports on lines 8 and 9 if they are not needed.